src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2909,21 +2908,21 @@ def cli_managementgroups_group_update_set(
 def cli_managementgroups_group_delete(cmd, client, group_name, no_register=False):
     if not no_register:
         _register_rp(cmd.cli_ctx)
-    return client.delete(group_name)
+    return client.begin_delete(group_name)
 
 
 def cli_managementgroups_subscription_add(
```

Comment:
@zhoxing-ms Question about this update method. For some reason when I try to update the group display name or parent nothing happens. I checked our API logs and the parameters are empty in the incoming request. Do I need to make any changes to the "update" calls when upgrading from 0.0.2 to 1.0.0? I didn't see anything on that sample PR you sent. Or perhaps something is just wrong with our python SDK. So far the changes to create, delete, show, and list all seem to be working and I just got stuck on update. Thanks!

Reply:
Could you show me the specific modified code link ?

Reply:
> Question about this update method. For some reason when I try to update the group display name or parent nothing happens. I checked our API logs and the parameters are empty in the incoming request. Do I need to make any changes to the "update" calls when upgrading from 0.0.2 to 1.0.0?

Actually, what I want to ask is which updating method do you mean? `cli_managementgroups_subscription_add` should be a creating method

Reply:
@zhoxing-ms I was talking about `cli_managementgroups_group_update_custom_func`, I did not mean to highlight `cli_managementgroups_subscription_add`

Reply:
@git-thomasdolan I pull the code from your banch and tried it. I found that `displayName` can update normally, but `parentGroupId` can't update.
```
Request URL: PATCH /providers/Microsoft.Management/managementGroups/zhoxing-test2?api-version=2021-04-01 HTTP/1.1
Request Body: {"displayName": "zhoxing2", "parentGroupId": "/providers/Microsoft.Management/managementGroups/zhoxing-test"}
x-ms-correlation-request-id: abd78d68-a349-4ac0-bd5a-6aa032874222
```

The request body sent conforms to the REST document  [management-groups/update](https://docs.microsoft.com/en-us/rest/api/managementgroups/management-groups/update), so this should be a REST service issue